dev-perl/Test-More-UTF8/Test-More-UTF8-0.05.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DIST_AUTHOR=MONS
+DIST_VERSION="0.05"
+
+inherit perl-module
+
+DESCRIPTION="Enhancing Test::More for UTF8-based projects"
+
+SLOT="0"
+LICENSE="|| ( Artistic GPL-1+ )"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~x86 ~ppc-macos"
+
+IUSE=""
+
```

Comment:
Upstream employs a bunch of harmful tests that should be exorcised with PERL_RM_FILES ( read perl-module.eclass docs for details ): https://st.aticpan.org/source/MONS/Test-More-UTF8-0.05/t

Those "pod" tests can only create trouble, never detect a real issue.